app/public/index.html

```diff
@@ -136,8 +136,8 @@
             </div>
         </div>
     </div>
-    <footer>
-        <div class="footer-disclaimer">
+    <footer class="footer" id="main-footer">
+        <div class="footer-disclaimer" id="footer-disclaimer">
```

Comment:
why did you introduce separate id for existing class? 
either merge into class or create a more meaningful name